src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -117,6 +130,34 @@ object VersionInjectorPlugin extends AutoPlugin {
         gitConfFile
     }
 
+  val stageChecksumTask = stageChecksum := {
+    import java.io.{ File => JFile }
+
+    def getFileList(dir: String): Seq[JFile] = new JFile(dir) match {
+      case f if f.exists && f.isDirectory => f.listFiles.filter(_.isFile).toSeq
+      case _ => List[JFile]()
```

Comment:
`Seq.empty` would be better (`empty` is clearer / nominally more efficient, and the type is inferrable).